ZPastel.Service/Repositories/IPastelRepository.cs

```diff
@@ -2,7 +2,7 @@
 using System.Threading.Tasks;
 using ZPastel.Model;
 
-namespace ZPastel.Persistence.Contract
+namespace ZPastel.Service.Repositories
```

Comment:
Create a new layer Core and moved the interfaces to this layer

Reply:
Fixed